constants.xml

```diff
@@ -228,6 +228,10 @@
         <PACKET_EPOCH_LATE_ALLOW>1</PACKET_EPOCH_LATE_ALLOW>
         <PACKET_BYTESIZE_LIMIT>1572864</PACKET_BYTESIZE_LIMIT>
         <SMALL_TXN_SIZE>1024</SMALL_TXN_SIZE>
+        <ACCOUNT_IO_BATCH_SIZE>2000000</ACCOUNT_IO_BATCH_SIZE>
+        <ENABLE_REPOPULATE>true</ENABLE_REPOPULATE>
+        <REPOPULATE_STATE_IN_DS>5</REPOPULATE_STATE_IN_DS>
```

Comment:
Put comment indicating REPOPULATE_STATE_IN_DS cannot be greater than REPOPULATE_STATE_PER_N_D

Reply:
Thanks, good point

Reply:
@kaikawaliu have u put the comment?